### PR TITLE
Add weight option

### PR DESCRIPTION
**Pull request by dbjpanda**
issue: https://github.com/d8-contrib-modules/tfa_basic/issues/16
2016-04-08 01_34_57-tfa settings _ drupal 8 sprint test page

### CODE REVIEW

**Comment by nerdstein** [comment]
Thanks @dbjpanda - does this correspond to one of the issues?

File: config/schema/tfa.schema.yml [hunk 1]
```diff
@@ -20,3 +20,6 @@ tfa.settings:
     validate_weights:
       type: sequence
       label: 'Validation plugins weights'
+    weight:
```
Comment (nerdstein) [inline config/schema/tfa.schema.yml:23]: Please annotate this as `tab_weight`, weight is too ambiguous here

File: src/Form/SettingsForm.php [hunk 2]
```diff
@@ -115,6 +115,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('enabled'),
       '#description' => t('Enable TFA for account authentication.'),
     );
+    
+	// Option to set tfa tab in userprofile
+	$form['weight'] = array(
```
Comment (nerdstein) [inline src/Form/SettingsForm.php:120]: Code formatting issues, please run PHPCS on this

Comment (nerdstein) [inline src/Form/SettingsForm.php:120]: Please rename all uses of "weight" to "tab_weight"

**Comment by dbjpanda** [comment]
Okay i will do it

File: src/Form/SettingsForm.php [hunk 3]
```diff
@@ -115,6 +115,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
+     '#type' => 'weight',
+     '#title' => $this->t('Weight of the tfa tab'),
```
Comment (therealssj) [inline src/Form/SettingsForm.php:122]: nit, 'Weight of TFA tab' instead of 'Weight of tfa tab'

**Pull request closed without merging**